target/linux/mvebu/image/cortexa9.mk

```diff
@@ -448,3 +464,20 @@ define Device/synology_ds213j
     -ppp -kmod-nft-offload -dnsmasq -odhcpd-ipv6only
 endef
 TARGET_DEVICES += synology_ds213j
+
+define Device/wd_cloud-gen2
+  DEVICE_VENDOR := Western Digital
+  DEVICE_MODEL := MyCloud Gen 2 (WD_Glacier)
+  DEVICE_PACKAGES += -uboot-envtools mkf2fs e2fsprogs badblocks \
+	partx-utils kmod-hwmon-drivetemp kmod-usb-storage \
+	kmod-fs-vfat fdisk
+  DEVICE_DTS := armada-375-wd_cloud-gen2
+  KERNEL := kernel-bin | append-dtb | uImage none
+  KERNEL_INSTALL := 1
+  KERNEL_SUFFIX := -uImage
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | uImage none
+  FILESYSTEMS := squashfs ext4
+  IMAGES := uRamdisk
+  IMAGE/uRamdisk := append-dtb | uRamdisk
```

Comment:
You can override parameters of `Build/uImage` by additional arguments.

`uImage gzip -T ramdisk -a 0xe00000 -e 0`

Reply:
And you can use `pad-to 4` or something for smaller dummy ramdisk instead of `append-dtb`.

Reply:
`IMAGE/uRamdisk := pad-to 8 | uImage gzip -T ramdisk -a 0xe00000 -e 0`
I'll do test tomorrow.

Reply:
So that did not work since this method use dd command to cut input file:
```
dd if=/home/user/openwrt-testflight/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/linux-mvebu_cortexa9/tmp/openwrt-mvebu-cortexa9-wd_cloud-gen2-squashfs-uRamdisk of=/home/user/openwrt-testflight/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/linux-mvebu_cortexa9/tmp/openwrt-mvebu-cortexa9-wd_cloud-gen2-squashfs-uRamdisk.new bs=8 conv=sync
dd: failed to open '/home/user/openwrt-testflight/build_dir/target-arm_cortex-a9+vfpv3-d16_musl_eabi/linux-mvebu_cortexa9/tmp/openwrt-mvebu-cortexa9-wd_cloud-gen2-squashfs-uRamdisk': No such file or directory
```
this one worked  since `dd` has something to chew on 

IMAGE/uRamdisk := append-dtb | pad-to 1 | uImage gzip -T ramdisk -a 0xe00000 -e 0

Reply:
update:
after looking into `pad-to`  usage this won't work like we need here. 
Something else must generate input file for uImage . For now settled on below:
`IMAGE/uRamdisk := append-dtb | | uImage gzip -T ramdisk -a 0xe00000 -e 0 -n 'DUMMY'`

Reply:
Oops, it's `pad-extra`, not `pad-to`... my mistake.